tools/topology/topology2/cavs-nocodec-bt.conf

```diff
@@ -113,130 +113,134 @@ Object.Dai {
 # Pipeline ID:1 PCM ID: 0
 Object.Pipeline {
 	# playback pipelines
-	passthrough-be.1 {
-		index	2
-		direction	playback
-		Object.Widget.pipeline.1 {
-			stream_name 'NoCodec-0'
-		}
-
-		Object.Widget.copier.1 {
-			dai_index 0
-			dai_type "SSP"
-			copier_type "SSP"
-			stream_name "NoCodec-0"
-			node_type $I2S_LINK_OUTPUT_CLASS
-		}
-	}
-
-	passthrough-playback.1 {
-		index	1
-		Object.Widget.pipeline.1 {
-			stream_name 'NoCodec-0'
-		}
-		Object.Widget.copier.1 {
-			stream_name 'SSP0 Playback'
-		}
-	}
+	passthrough-be [
```

Comment:
The changes to these new pipeline object instantiations with number of pipeline arrays were hardest to follow. The new jumping indexes do not look the most logical choices, but then again I can see that they come from history. In the end the indexes do not matter for anything else but they are needed when referring to these objects from outside, do they?

Reply:
yes, the indices have no meaning in most cases except for the widget names. But I havent touched those obects within pipeline class definitions yet. That will be the next target after this one is merged.